### ISSUE TITLE
"Handy tools" contains different types of things

### ISSUE THREAD

**Issue by konhi** [kmaasrud/awesome-obsidian#38]
**Handy tools** section contains different type of tools:
- **Plugins**, e.g. Obsidian Day Planner
- **External Tools**, e.g. Oboe
- **Templates**, e.g. Obsidian Starter Templates

I think it should be split for clarity! 😀

---

I think another issue is that there is a credits section, but these are open-source projects with multiple contributors. Should credits then stay in the table? Repository is linked anyway, so user can see contributors anyway. Let me know what do you think! 😊

**Comment by kmaasrud** [comment]
I agree with the splitting you have done in #39. I made that section before the Obsidian API was released and we did not have many plugins/other tools, so it made sense. But now this is a much better solution!

I like the "community" feeling we get when giving credits, so I think we should keep them for now. Or at least as long as the contributor lists don't get too long - in which case I might consider removing them.
